subsys/infinispan/src/main/java/org/commonjava/indy/subsys/infinispan/CacheProducer.java

```diff
@@ -101,10 +106,13 @@ public void start()
     {
         startRemoteManager();
         startEmbeddedManager();
+        startClusterManager();
     }
 
     private RemoteCacheManager remoteCacheManager;
 
+    private EmbeddedCacheManager clusterCacheManager;
```

Comment:
I'm confused about why we need a separate EmbeddedCacheManager for clustered caches. It seems like an infinispan.xml can contain multiple cache-container entries, and multiple different jgroups configuration file references...with the transports in the cache containers referencing 0..1 of those jgroups configs. Can we not configure it all in a single infinispan.xml and just allocate the different caches we use to different cache containers in that config file?

Reply:
Yeah, that'd be better, but when i tried to do that earlier, it reported the invalid element error, and i checked the config xsd https://docs.jboss.org/infinispan/9.4/configdocs/infinispan-config-9.4.html,  looks it does not allow the multiple cache-containers.

Reply:
Okay, so that's not going to work. Your idea is a good one in that case.